src/OpenLoco/src/Ui/Windows/StationWindow.cpp

```diff
@@ -787,6 +782,442 @@ namespace OpenLoco::Ui::Windows::Station
         }
     }
 
+    // NB: This namespace shares a fair bit of code with the VehicleList window.
```

Comment:
have you marked where it is different or is it a little bit everywhere.

Reply:
It's a little bit everywhere, yeah. Doesn't check for company/owner id or cargo, and instead checks station ids. Shows less information (just vehicle status), that kind of thing. Not thrilled about how much is copied, though.